content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -328,6 +584,96 @@ Confirm that roughly 10% traffic from the 100 requests go to reviews-v2:
 $ kubectl exec deploy/sleep -- sh -c "for i in \$(seq 1 100); do curl -s http://istio-ingressgateway.istio-system/productpage | grep reviews-v.-; done"
 {{< /text >}}
 
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+Apply the reviews HTTPRoute to control 90% traffic to reviews v1 and 10% traffic to reviews v2.
+
+{{< text bash >}}
+$ kubectl apply -f @samples/bookinfo/networking/virtual-service-reviews-90-10.yaml@
+$ kubectl apply -f @samples/bookinfo/networking/destination-rule-reviews.yaml@
+{{< /text >}}
+
+Confirm that roughly 10% traffic from the 100 requests go to reviews-v2:
+
+{{< text bash >}}
+$ kubectl exec deploy/sleep -- sh -c "for i in \$(seq 1 100); do curl -s http://bookinfo-gateway-istio.istio-system/productpage | grep reviews-v.-; done"
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+Create subsets of the reviews Service, so we can control traffic to v1 and v2 versions of reviews:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: reviews-v1
+  labels:
+    app: reviews
+    service: reviews
+spec:
+  ports:
+    - port: 9080
+      name: http
+  selector:
+    app: reviews
+    version: v1
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: reviews-v2
+  labels:
+    app: reviews
+    service: reviews
+spec:
+  ports:
+    - port: 9080
+      name: http
+  selector:
+    app: reviews
+    version: v2
+EOF
+{{< /text >}}
+
+Create an HTTPRoute to control 90% traffic to reviews v1 and 10% traffic to reviews v2.
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: reviews
+spec:
+  parentRefs:
+  - kind: Service
+    name: reviews
+  rules:
+  - backendRefs:
+    - name: reviews-v1
+      port: 9080
+      weight: 90
+    - name: reviews-v2
+      port: 9080
+      weight: 10
```

Comment:
@frankbu do you prefer I add ^ to `samples/bookinfo/gateway-api` instead of inlining?

Reply:
Yes, the plan for `samples/bookinfo/gateway-api` was to add more yaml files as needed by docs, rather than making mirror copies of everything in `samples/bookinfo/networking`.

Reply:
@frankbu I created https://github.com/istio/istio/pull/44097 to add ^ manifest to Istio.